.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.19.3
```

Comment:
Is there any preference for this over the current version specified in https://github.com/supabase/cli/blob/main/go.mod#L3?